examples/conanfile/package_id/settings_and_options.rst

```diff
@@ -0,0 +1,206 @@
+.. _examples_conanfile_package_id_settings_options:
+
+How settings and options of a recipe influence its package ID
```

Comment:
Should we define some convention to name package-id? package ID, package-id, package_id?

Reply:
I have always thought about it as "the ID of the package", so I noramlly use just "package ID" (although maybe "package's ID" is the most correct one? I don't like it :smile:).  Anyway, "package-id" or "package ID" will be fine for me 👍

Reply:
👍 for plain package id